### PR TITLE
VACMS-1269 homepage improvements

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #1269. Builds on the last login PR, further styles the PIV login container & adds a toggle to swap between the two login flows, while staying at the same URL. Figma this is based on is here: https://www.figma.com/file/w0z1NIdY7nent0Q6r7hjTD/Login-page?node-id=330%3A64

Added a footer to the **login page only** that mimics the feel of design.va.gov and developer.va.gov. Consistency builds trust!

Still need to add styles for the "new look" block, but wasn't sure how that would be done in the current system (is there an existing block?) so left it alone for this PR

## Testing done
Lots of screen size adjustments to get spacing feeling okay on mobile/tablet in addition to the larger screens. User/pass still works when hidden... I assume PIV login still works, can't use it locally.

Fixed WAVE errors and warnings.... I looked at it with VoiceOver and with the chromevox extension and both seemed to be navigable... can probably still be improved, but it works for now.

## Screenshots
extra wide:
Screen Shot 2021-12-22 at 4 15 55 PM
Screen Shot 2021-12-22 at 4 16 05 PM

tablet:
Screen Shot 2021-12-22 at 4 16 37 PM
Screen Shot 2021-12-22 at 4 16 47 PM

## QA steps
visit tugboat env, see that the unauth page is updated and the toggle works for switching between login flows.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

Claro only at this point, no announcement needed.

### CODE REVIEW

**Review by swirtSJW — DISMISSED** [review]
Looks good Tanner.  I think the current test failures are unrelated.   I'll trigger a rebuild and see.

**Comment by tjheffner** [comment]
@swirtSJW the new commits don't change too much, some minor a11y cleanup but nothing visual (thanks WAVE!)... Updated the original description to mention screenreaders. I think I'm done tinkering on this page after those, pending any design feedback🤞

File: docroot/themes/custom/vagovclaro/templates/layout/footer.html.twig [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * @file
+ * VA.gov Claro implementation of VA footer. Appearance should be relatively
+ * consistent with other VA digital properties.
+ */
+#}
+
+<footer class="va-footer">
+  <div class="layout-container">
+    {# these links should be tied to a menu imo #}
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovclaro/templates/layout/footer.html.twig:11]: Agreed.  Good idea.

**Review by swirtSJW — DISMISSED** [review]
All looks good.   Looks like we have a conflict or two. that need to be resolved.

File: docroot/themes/custom/vagovclaro/templates/layout/footer.html.twig (hunk 1 above)
Comment (kevwalsh) [reply docroot/themes/custom/vagovclaro/templates/layout/footer.html.twig:11]: I created #7420 as a followup.

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**